.github/workflows/test.yml

```diff
@@ -223,7 +223,7 @@ jobs:
           secret: ${{ secrets.BUILDPULSE_SECRET_ACCESS_KEY }}
 
       - uses: actions/upload-artifact@v3
-        if: always()
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
Are you sure this is correct? The other `if` aboe is without the `{{ .... }}` and I can never remember the difference and meaning.

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
If I read correctly, the one [above](https://github.com/prisma/prisma/pull/17979/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R216) looks like it would be evaluated as a string?

Reply:
The docs confuse me even more, each time: 
<img width="598" alt="image" src="https://user-images.githubusercontent.com/183673/219825945-1738d5e6-f222-433a-b48f-57f078ea1878.png">

Reply:
Ok, that settles the debate: both work

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information about if conditionals, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif)."

Reply:
Best not to have mentioned it, ever :D

Reply:
Fixed it.